metagpt/provider/base_llm.py

```diff
@@ -146,6 +146,8 @@ async def aask(
             message.append(self._user_msg(msg, images=images))
         else:
             message.extend(msg)
+        if stream is None:
+            stream = self.config.stream
```

Comment:
If modified in this way, the default stream will also become False due to the setting in llm_config. I recommend changing the setting in llm_config to the default True to maintain consistency

Reply:
Oh Im sorry. I should be more carefully。I should have thought of that。